README.md

```diff
@@ -108,26 +244,53 @@ class Value:
         self.value = value
 
     def __str__(self):
-        return str(self.value)
+        return self.value
 
 
-class Enumeration(Enum):
+class CustomObjectEnum(Enum):
     A = Value(1)
-    B = Value('foo')
+    B = Value('B')
 
 
 class SomeModel(models.Model):
-    enumeration = EnumChoiceField(Enumeration)
+    enumerated_field = EnumChoiceField(enum_class=CustomObjectEnum)
 ```
 
 We'll have the following:
 
-* `SomeModel.enumeration.choices == (('1', '1'), ('foo', 'foo'))`
-* `SomeModel.enumeration.max_length == 3`
+* `SomeModel.enumerated_field.choices == (('1', '1'), ('B', 'B'))`
+* `SomeModel.enumerated_field.max_length == 3`
+
+## Using Python's `enum.auto`
```

Comment:
Lets also add an example with auto for string values (as described in Python's documentation)